### ISSUE TITLE
Photom fails with circular import in 1.7.2

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#7275]
_Issue [JP-2934](https://jira.stsci.edu/browse/JP-2934) was created on JIRA by [Bryan Hilbert](https://jira.stsci.edu/secure/ViewProfile.jspa?name=hilbert):_

When I try to import the photom step under 1.7.2, I get an error about a potential circular import:

```
In [2]: import jwst

In [3]: jwst.*_version_*
Out[3]: '1.7.2'

In [1]: from jwst.photom import PhotomStep
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-e2049d242273> in <module>
----> 1 from jwst.photom import PhotomStep

~/miniconda3/envs/jwst-py3.8/lib/python3.8/site-packages/jwst/photom/*_init_*.py in <module>
----> 1 from .photom_step import PhotomStep
      2 
      3 *_all_* = ['PhotomStep']

~/miniconda3/envs/jwst-py3.8/lib/python3.8/site-packages/jwst/photom/photom_step.py in <module>
      2 
      3 from ..stpipe import Step
----> 4 from . import photom
      5 from .. import datamodels
      6 

~/miniconda3/envs/jwst-py3.8/lib/python3.8/site-packages/jwst/photom/photom.py in <module>
      8 from .. import datamodels
      9 from .. datamodels import dqflags
---> 10 from .. lib.wcs_utils import get_wavelengths
     11 
     12 log = logging.getLogger(*_name_*)

~/miniconda3/envs/jwst-py3.8/lib/python3.8/site-packages/jwst/lib/wcs_utils.py in <module>
      1 import numpy as np
----> 2 from ..assign_wcs import niriss
      3 
      4 
      5 WFSS_EXPTYPES = ['NIS_WFSS', 'NRC_WFSS', 'NRC_GRISM', 'NRC_TSGRISM']

~/miniconda3/envs/jwst-py3.8/lib/python3.8/site-packages/jwst/assign_wcs/*_init_*.py in <module>
----> 1 from .assign_wcs_step import AssignWcsStep
      2 from .nirspec import (nrs_wcs_set_input, nrs_ifu_wcs, get_spectral_order_wrange)
      3 from .niriss import niriss_soss_set_input
      4 from .util import update_fits_wcsinfo
      5 

~/miniconda3/envs/jwst-py3.8/lib/python3.8/site-packages/jwst/assign_wcs/assign_wcs_step.py in <module>
      4 from ..lib.exposure_types import IMAGING_TYPES
      5 import logging
----> 6 from .assign_wcs import load_wcs
      7 from .util import MSAFileError, update_fits_wcsinfo
      8 

~/miniconda3/envs/jwst-py3.8/lib/python3.8/site-packages/jwst/assign_wcs/assign_wcs.py in <module>
      6 from ..lib.exposure_types import IMAGING_TYPES, SPEC_TYPES, NRS_LAMP_MODE_SPEC_TYPES
      7 from ..lib.dispaxis import get_dispersion_direction
----> 8 from ..lib.wcs_utils import get_wavelengths
      9 
     10 log = logging.getLogger(*_name_*)

ImportError: cannot import name 'get_wavelengths' from partially initialized module 'jwst.lib.wcs_utils' (most likely due to a circular import) (/Users/hilbert/miniconda3/envs/jwst-py3.8/lib/python3.8/site-packages/jwst/lib/wcs_utils.py)
```

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-2934?focusedCommentId=609981#comment-609981):_

Confirmed that I get the same error in the 1.7.3dev environment.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-2934?focusedCommentId=614522#comment-614522):_

Fixed by #7330

**Comment by stscijgbot-jp** [comment]
_Comment by [Misty Cracraft](https://jira.stsci.edu/secure/ViewProfile.jspa?name=cracraft) on [JIRA](https://jira.stsci.edu/browse/JP-2934?focusedCommentId=628421#comment-628421):_

Since the pipeline is several versions beyond this one, and version 1.9.4 that I just tested shows no sign of this issue, can we close the ticket?
